xbmc/pvr/recordings/PVRRecording.h

```diff
@@ -82,8 +84,23 @@ namespace PVR
     std::string   m_strFanartPath;    /*!< fanart path */
     unsigned      m_iRecordingId;     /*!< id that won't change while xbmc is running */
 
+  private:
     CPVRRecording(void);
     CPVRRecording(const PVR_RECORDING &recording, unsigned int iClientId);
+
+    // Prevent copy construction, even for CPVRRecording instances and friends.
+    // Note: Only declared, but intentionally not implemented
+    //       to prevent compiler generated copy ctor and to force
+    //       a linker error in case somebody tries to call it.
+    CPVRRecording(const CPVRRecording &tag);
```

Comment:
just implement them in the private part, then nobody can use it anyway. except when you use friend class hacks ;-)
every singleton in the code does it like this.